examples/user-openjdk/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM smebberson/alpine-openjdk
```

Comment:
This doesn't need pinning, as it will only ever be built in development/testing.